open-metadata-implementation/common-services/generic-handlers/src/main/java/org/odpi/openmetadata/commonservices/generichandlers/RelationalDataHandler.java

```diff
@@ -2265,25 +2265,14 @@ public String createDatabaseTableForSchemaType(String               userId,
                                                                          repositoryHelper);
         }
 
-        /*
-         * The schema type that connects the database schema asset to the tables has been created/retrieved.
-         * Now work out the position of the new table in the database schema type.  This is used to set the element position.
-         * Since this value begins with 0 as the first element, the table count is this table's position.
-         */
-        int tableCount = databaseTableHandler.countSchemaAttributes(userId,
```

Comment:
Looks like effectiveTime was used in here in countSchemaAttributes call. Not sure if it is better to remove it from the method signature (then all the calls to this will need to be changed as well) or we dismiss codeQL alert.